srtcore/queue.cpp

```diff
@@ -1215,7 +1215,7 @@ void* CRcvQueue::worker(void* param)
        // Check connection requests status for all sockets in the RendezvousQueue.
        // Pass the connection status from the last call of:
        // worker_ProcessAddressedPacket --->
-       // worker_TryAsyncRend_OrStore --->
+       // worker_TryAsyncSend_OrStore --->
```

Comment:
That wasn't a typo. The `Rend` part is a shortcut for `Rendezvous`.

Reply:
Will leave the naming until better time then.